.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Acme e2e Test CI
+
+on:
```

Comment:
Can we run this for every pull request?

Reply:
Should already be in: this is how it looks like once pulled in:
https://github.com/srose/keycloak-project-example/pull/3/checks